src/components/Layout/Contact.tsx

```diff
@@ -13,73 +13,145 @@ import {
   Text,
   Grid,
   GridItem,
-  Textarea
+  Textarea,
+  useToast
 } from "@chakra-ui/react";
-
+import { sendEmail } from "../../utils/fetchData";
+import { error } from "console";
 const Contact = () => {
+  const toast = useToast();
+  const handleSendEmail = (e: any) => {
+    e.preventDefault();
+    const data = new FormData(e.target);
+    const dataObject = Object.fromEntries(data.entries());
+    sendEmail(dataObject)
+      .then(response => {
+        console.log(response);
+        toast({
+          title: "",
+          description: "",
+          status: "success",
+          duration: 3000,
+          isClosable: true,
+          position: "top",
+          render: () => (
+            <>
+              <Box p="3" bg="green">
+                {response.data.message}
+              </Box>
+            </>
+          )
+        });
+        e.target.reset();
+      })
+
+      .catch(error => {
+        console.log(error);
```

Comment:
See comment above - will want to remove this

Reply:
removed